### PR TITLE
php-vcr issue #67: unit test fail on Windows 7.

### PR DESCRIPTION
**Pull request by b01**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <phpunit bootstrap="tests/bootstrap.php"
```
Comment (dbu) [inline phpunit.xml]: the phpunit.xml file should be renamed to phpunit.xml.dist and phpunit.xml be in .gitignore. then its much easier to customize the phpunit file.

File: phpunit.xml [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
-         colors="true"
```
Comment (dbu) [inline phpunit.xml]: that sounds like a phpunit bug - if it can't output colors it should ignore this. what version of phpunit do you have?

**Comment by renatomefidf** [comment]
@b01 sorry for the delay, sadly due to that your PR got outdated. Are you still willing to work on this?
If yes maybe you could keep the colors true on the phpunit config. I'll check better the PR if you want to proceed!

Thank you for your time and sorry again for the delay.

**Comment by renatomefi** [comment]
Since no one else complained about it and the author is not responding I'm closing this for now.
If anyone wants to make it again please fell free to open a new PR!

Thank you all

**Pull request closed without merging**